cms/djangoapps/contentstore/views/organization.py

```diff
@@ -5,8 +5,7 @@
 from django.views.generic import View
 
 from openedx.core.djangolib.js_utils import dump_js_escaped_json
-from util.organizations_helpers import get_organizations
```

Comment:
nit: only import line should have changed

Reply:
`get_organizations` is not been used in file so thats why I remove it.

Reply:
Ok